nginx.rb

```diff
@@ -17,6 +17,7 @@
 set_default(:domain_name, "#{nginx_config_name}")
 set_default(:nginx_access_log, "#{nginx_shared_path}/log/nginx_#{nginx_config_name}_access.log")
 set_default(:nginx_error_log, "#{nginx_shared_path}/log/nginx_#{nginx_config_name}_error.log")
+set_default(:nginx_proxy_read_timeout, 75)
```

Comment:
Sorry, for not catching it first time, and repeating corrections.

But, actually setting default to 75 seconds is not good for most Rails applications. Capistrano recipes are generic recipes that are supposed to work for various rails apps.

Because of this I'd prefer to have set default to some reasonable value, something more typical for web application like 30 seconds, and set specific value in Coiney rails configuration, i.e. in Capiche config section.